### PR TITLE
Add propose remove

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

File: cli/multisig.go [hunk 1]
```diff
@@ -504,6 +505,84 @@ var msigApproveCmd = &cli.Command{
...
+
+		fmt.Println("sent remove proposal in message: ", msgCid)
+
+		wait, err := api.StateWaitMsg(ctx, msgCid, build.MessageConfidence)
+		if err != nil {
+			return err
+		}
+
+		if wait.Receipt.ExitCode != 0 {
+			return fmt.Errorf("add proposal returned exit %d", wait.Receipt.ExitCode)
```
Comment (arajasek) [inline cli/multisig.go:572]: "add proposal" is wrong

File: cli/multisig.go [hunk 2]
```diff
@@ -504,6 +505,84 @@ var msigApproveCmd = &cli.Command{
...
+			}
+			from = f
+		} else {
+			defaddr, err := api.WalletDefaultAddress(ctx)
+			if err != nil {
+				return err
+			}
+			from = defaddr
+		}
+
```
Comment (arajasek) [inline cli/multisig.go:558]: It'd be nice to add a check at this stage that the `from` we're gonna use is a valid signer...This isn't specific to this PR, though, we can just crack an issue for it for now.

**Review by arajasek — COMMENTED** [review]
Do we need a `remove-approve` (and `remove-cancel`) too?

**Review by whyrusleeping — APPROVED** [review]

File: cli/multisig.go (hunk 2 above)
Comment (whyrusleeping) [reply cli/multisig.go:558]: Would be nice to do, for sure.

**Comment by whyrusleeping** [comment]
> Do we need a remove-approve (and remove-cancel) too?

No, we don't need any of those methods, just use `approve`.

**Comment by arajasek** [comment]
> > Do we need a remove-approve (and remove-cancel) too?
> 
> No, we don't need any of those methods, just use `approve`.

Note that we don't have a generic `cancel` command.

**Pull request merged**